types/wlr_buffer.c

```diff
@@ -64,6 +64,13 @@ bool wlr_buffer_get_dmabuf(struct wlr_buffer *buffer,
 	return buffer->impl->get_dmabuf(buffer, attribs);
 }
 
+bool wlr_buffer_get_data_ptr(struct wlr_buffer *buffer, void **data,
+	uint32_t *size) {
```

Comment:
Style: need one extra indentation level for this continuation line, otherwise we can't tell it apart from the function body.